spec/javascript/components/supplemental_application/sections/PreferencesTable.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import PreferencesTable from '~/components/supplemental_application/sections/PreferencesTable'
+import application from '../../../fixtures/domain_application'
+
+const applicationMembers = [{id: 'a0n0x000000AbE6AAK', first_name: 'karen', last_name: 'jones'}]
+const fileBaseUrl = 'https://test.force.com'
+const onSave = () => {}
+const onDismissError = () => {}
+const formApi = {}
+
+describe('PreferencesTable', () => {
+  test('should render a table of only preferences that the application receives (Receives_Preference = true) ', () => {
+    // const context = cloneDeep(baseContext)
+    const component = renderer.create(
+      <PreferencesTable
+        application={application}
+        applicationMembers={applicationMembers}
+        onSave={onSave}
+        fileBaseUrl={fileBaseUrl}
+        onPanelClose={onDismissError}
+        formApi={formApi}
+      />
+    )
+
+    let tree = component.toJSON()
+    expect(tree).toMatchSnapshot()
```

Comment:
@chrisdolendo we're trying to avoid adding more tests like this where we're just checking that a snapshot matches because it's so hard to actually check that the snapshot contains what you expect. Can you update this test to explicitly check that your expected preferences are in there and the one's that you're not expecting to be in there aren't in there?

Reply:
Done!